__mocks__/got.js

```diff
@@ -0,0 +1,73 @@
+const people = require('./../assets/people.json');
+const films = require('./../assets/films.json');
+const species = require('./../assets/species.json');
+const planets = require('./../assets/planets.json');
+
+
+const got = (url, options) => {
+    let data, isError;
+    if (/^https:\/\/swapi.co\/api\/*/.test(url)) {
+
+        const path = url.replace('https://swapi.co/api/', '').split('/');
+        const pathNum = parseInt(path[1])-1;
+
+        if (isNaN(pathNum) || pathNum > 7) {
```

Comment:
This is not reusable, and is not easily extensible.

Why would you want to find data by the index that it appears in the test data? What happens when that changes? Are you really going to expect someone to go through and update all the tests?

Show me how you could do this better to make it more extensible for future test cases that might come up

Reply:
I am not sure about this one, but I kept pathNum(objectId) restricted less than 7 because I am allowing test only for first 2 people. 

Should we put all 87 people in the json file?